atat_provisioning_wizard_api.yaml

```diff
@@ -961,73 +977,114 @@ components:
+      required:
+        - clin_number
+        - idiq_clin
+        - total_clin_value
+        - obligated_funds
+        - pop_start_date
+        - pop_end_date
     ApplicationStep:
       description: Represents step 3 of the Portfolio Draft Wizard
       type: object
       properties:
         applications:
           type: array
           items:
-            type: object
-            properties:
-              name:
-                type: string
-              description:
-                type: string
-              environments:
-                type: array
-                items:
-                  type: object
-                  properties:
-                    name:
-                      type: string
-                    operators:
-                      type: array
-                      items:
-                        type: object
-                        properties:
-                          first_name:
-                            type: string
-                          last_name:
-                            type: string
-                          email:
-                            type: string
-                            format: "email"
-                          access:
-                            type: string
-                            description: Operator Access Level to Environment
-                            enum:
-                              - administrator
-                              - read_only
+            $ref: '#/components/schemas/ApplicationObject'
+    ApplicationObject:
+      description: Represents an Application in a Portfolio
+      type: object
+      additionalProperties: false
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        environments:
+          type: array
+          items:
+            $ref: '#/components/schemas/EnvironmentObject'
+      required:
+        - name
+        - description
+        - environments
+    EnvironmentObject:
+      description: Represents an Environment for a specific Application
+      type: object
+      additionalProperties: false
+      properties:
+        name:
+          type: string
+        operators:
+          type: array
+          items:
+            $ref: '#/components/schemas/OperatorObject'
+      required:
+        - name
+        - operators
+    OperatorObject:
+      description: Represents an Operator of an Environment for a specific Application
+      type: object
+      additionalProperties: false
+      properties:
+        first_name:
+          type: string
+        last_name:
+          type: string
+        email:
+          type: string
+          format: "email"
+        access:
+          description: Operator Access Level to Environment
+          type: string
+          enum:
+            - administrator
+            - read_only
+      required:
+        - first_name
+        - last_name
+        - email
+        - access
     PortfolioDraft:
```

Comment:
Do we want to add 'additionalProperties: false' to the remaining schema objects?  PortfolioDraft, PortfolioDraftSummary, etc.

Reply:
Added `additionalProperties: false` to all schema Objects. If there is some reason not include it on certain objects let me know.